File: Deal/src/main/java/ru/leonov/deal/model/record/PassportRecord.java
package ru.leonov.deal.model.record;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

/**
 * POJO that contain information about client's passport.
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class PassportRecord {

    /**
     * Client's passport series.
     */
    private Integer series;

    /**
     * Client's passport number.
     */
    private Integer number;

    /**
     * Client's passport issue date.
     */
    private LocalDate issueDate;

    /**
     * Name of client's passport issue organization.
     */
    private String issueBranch;
}

File: Deal/deal_impl/src/main/kotlin/ru/leonov/deal/model/record/PassportRecord.kt
package ru.leonov.deal.model.record

import java.time.LocalDate

/**
 * POJO that contain information about client's passport.
 */
data class PassportRecord (
    /**
     * Client's passport series.
     */
    var series: Int,
    /**
    * Client's passport number.
    */
    var number: Int,
    /**
    * Client's passport issue date.
    */
    var issueDate: LocalDate? = null,
    /**
     * Name of client's passport issue organization.
     */
    var issueBranch: String? = null
)
